### DISCUSSION TITLE
Announcing Serwist's migration to GitLab

### DISCUSSION THREAD

**Discussion by DuCanhGH** [serwist/serwist#116] [Announcements]
Serwist has migrated to GitLab.

This was the result of GitHub flagging my account, organizations, and repositories as spam. GitHub has resolved the issue since then, but by the time they did that, the trust had already been broken, and the process had already completed. I, frankly, don't take this kind of issue lightly, for I fear that something like this may happen again in the future.

In the future, Serwist will be developed at GitLab and mirrored to GitHub. Issues and discussions opened at GitHub will still be supported, but pull requests will be closed.

Here is [Serwist's new home](https://gitlab.com/serwist/serwist).

Thank you!

**Comment by bestickley** [comment]
The above is not true anymore, correct?

**Reply by DuCanhGH** [reply]
@bestickley yes, that's true. This was mentioned in [the introduction of `serwist`](https://github.com/serwist/serwist/discussions/118).

**Reply by bestickley** [reply]
Thank you for clarifying! :)
